.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,7 @@
+- id: julia-formatter
+  name: "Julia Formatter"
+  entry: "./bin/format.jl ."
```

Comment:
this file shouldn't be used it's super old

Reply:
Ah, okay. Reverted to `julia -e ...`. 🙂